ruka:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Position Controllers ---------------------------------------
  joint_1_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_1
    pid: {p: 100.0, i: 0.0, d: 15.0}
  joint_2_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_2
    pid: {p: 75.0, i: 0.0, d: 4.5}
  joint_3_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_3
    pid: {p: 2.5, i: 0.0, d: 0.45}
  joint_4_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_4
    pid: {p: 1.0, i: 0.0, d: 0.06}
